daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1204,15 +1204,15 @@ private[lf] object SBuiltin {
                   true
                 case SKeyLookupResult.NotFound | SKeyLookupResult.NotVisible =>
                   onLedger.ptx = onLedger.ptx.copy(keys = onLedger.ptx.keys + (gkey -> None))
-                  machine.tryHandleException()
+                  machine.tryHandleSubmitMustFail()
               },
             )
           )
       }
     }
   }
 
-  /** $getTime :: Token -> Timestamp */
+  /** $getTime :: Token -> Update Timestamp */
```

Comment:
IIRC `get_time` is also used in Scenario. Is it the same speedy builtin?

Reply:
No, it isn’t. it’s handled via the free monad.

Reply:
Scenario uses a free monad now? This is news to me!